app/models/revision.rb

```diff
@@ -4,21 +4,18 @@ class Revision < ApplicationRecord
 
   def serializable_hash(options = {})
     allowed_options = [
-      'uuid',
-      'items_key_id',
-      'duplicate_of',
-      'enc_item_key',
-      'content',
-      'content_type',
       'auth_hash',
-      'deleted',
```

Comment:
`deleted` doesn't appear to be an actual field on revisions, not sure how it ended up here.